MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+
+### Using IDataFormSourceProvider
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public string Country { get; set; }
+    
+    public class DataFormItemsSourceProvider : IDataFormSourceProvider
+    {
+         public override object GetSource(string sourceName)
+        {
+            var list = new List<string>();
+            if (sourceName == "Country")
+            {
+                list.Add("India");
+                list.Add("USA");
+                list.Add("China");
+                list.Add("Italy");
+            }
+            return list;
+        }
+    }
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+    dataForm.RegisterEditor("ItemName", DataFormEditorType.Picker);
+
+{% endhighlight %}
+{% endtabs %}
+
+### Using event
+
+You can also set `ItemsSource` for picker editor by using the [ItemsSource]`Link` property in the [DataFormPickerItem]`Link`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name" && e.DataFormItem is DataFormPickerItem pickerItem)
+        {
+            var list = new List<string>();
+            list.Add("Home");
+            list.Add("Food");
+            list.Add("Utilities");
+            list.Add("Education");
+            pickerItem.ItemsSource = list;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Loading complex type property values in picker
+
+You can display the complex type property values in picker editor by using the [GetSource]`Link` override method of IDataFormSourceProvider class, which is used to get source list as complex property values for picker editor and set it to `IDataFormSourceProvider` property of SfDataForm. You need to use `GenerateDataFormItem`event to set [DisplayMemberPath]`Link` and [ValueMemberPath]`Link` property value DataFormPickerItem for complex type property.
+
+N> Class cannot be directly set as data type for picker editor in this complex type scenario.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+    dataForm.RegisterEditor("City", DataFormEditorType.Picker);
+    dataForm.DataObject = new DataModel();
+        dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+ 
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "City" && e.DataFormItem is DataFormPickerItem pickerItem)
+        {
+            pickerItem.DisplayMemberPath = "City";
+            pickerItem.ValueMemberPath = "PostalCode";
+        }
+    } 
+ 
+    public class DataFormItemsSourceProvider : IDataFormSourceProvider
```

Comment:
Use one real time use for complex item for all editors

Reply:
code added